pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -317,6 +317,43 @@ public void testTimeFunc()
     assertEquals(todayStr, expectedTodayStr);
   }
 
+  @Test
+  public void testLiteralOnlyFunc()
+      throws Exception {
+    long currentTsMin = System.currentTimeMillis();
+    String sqlQuery = "SELECT 1, now() as currentTs, 'abc', toDateTime(now(), 'yyyy-MM-dd z') as today, now()";
+    JsonNode response = postSqlQuery(sqlQuery, _brokerBaseApiUrl);
+    long currentTsMax = System.currentTimeMillis();
+
+    assertEquals(response.get("resultTable").get("dataSchema").get("columnNames").get(0).asText(), "1");
+    assertEquals(response.get("resultTable").get("dataSchema").get("columnNames").get(1).asText(), "currentTs");
+    assertEquals(response.get("resultTable").get("dataSchema").get("columnNames").get(2).asText(), "abc");
+    assertEquals(response.get("resultTable").get("dataSchema").get("columnNames").get(3).asText(), "today");
+    String nowColumnName = response.get("resultTable").get("dataSchema").get("columnNames").get(4).asText();
```

Comment:
Ideally the column name should be `now()` right?
This is not critical, but better put a TODO?

Reply:
Will add a TODO